mws/mws.py

```diff
@@ -75,17 +75,33 @@ def calc_request_description(params):
       "bar=4&baz=potato&foo=1"
     """
     description_items = []
+    params = clean_extra_data(params)
```

Comment:
This is called twice at the moment (see other note).

This call can be removed. I think to properly test param outputs, we need to keep the other call, but move it slightly.